ivy/functional/ivy/experimental/activations.py

```diff
@@ -194,8 +212,13 @@ def thresholded_relu(
 @to_native_arrays_and_back
 @handle_array_function
 @handle_device_shifting
+@handle_complex_input
 def relu6(
```

Comment:
Complex numbers are marked as unsupported for `relu6` in the `tf` and `torch` backends. The `paddle` backend has a custom implementation for `relu6` on complex numbers which is now obsolete and should be removed

Reply:
Done.